databases/week1/week1.sql

```diff
@@ -0,0 +1,67 @@
+
+-- Find out how many tasks are in the task table 
+
+SELECT count(id) 
+AS total_tasks
+FROM TASK;
+
+--  Find out how many tasks in the task table do not have a valid due date;
+
+SELECT count(id) 
+AS Invalid_Dates_Count
+FROM TASK
+where due_date IS null ;   -- output 8
+
+--  Find all the tasks that are marked as done
+
+SELECT task.title,status.name
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name= 'Done';
+
+-- Find all the tasks that are not marked as done
+
+SELECT task.title,status.name
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name= 'Not started' OR status.name= 'In progress' ;
```

Comment:
```suggestion
SELECT task.title,status.name
FROM task, status
INNER JOIN status ON task.status_id = status.id
WHERE status.name= 'Not started' OR status.name= 'In progress' ;
```

Reply:
Hi Prital
i am getting this error
Error Code: 1066. Not unique table/alias: 'status'